src/client/index.js

```diff
@@ -211,10 +211,8 @@ export async function signIn(provider, options = {}, authorizationParams = {}) {
       callbackUrl,
       json: true,
     }),
-  }
+  })
```

Comment:
This change was to check the new tooling worked, happy to revert it if you think before it read better 👍🏽